main/exercise/question.class.php

```diff
@@ -1938,16 +2133,16 @@ public static function saveQuestionOption($question_id, $name, $course_id, $posi
 
     /**
      * @param int $question_id
-     * @param int $course_id
+     * @param int $courseId
      */
-    public static function deleteAllQuestionOptions($question_id, $course_id)
+    public static function deleteAllQuestionOptions($question_id, $courseId)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The variable $question_id is not named in camelCase.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5018478)